jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -601,26 +601,52 @@ properties:
             fits_keyword: EXP_TYPE
             blend_table: True
           start_time:
-            allOf:
-              - type: number
-              - fits_keyword: EXPSTART
-                title: UTC exposure start time
-                blend_table: True
-                blend_rule: first
+            title: UTC exposure start time
+            type: number
+            fits_keyword: EXPSTART
+            blend_table: True
+            blend_rule: first
+          start_time_mjd:
+            title: "[d] exposure start time in MJD"
+            type: number
+            fits_keyword: MJD-BEG
+            fits_hdu: SCI
+            blend_table: True
+            blend_rule: first
           mid_time:
-            allOf:
-              - type: number
-              - fits_keyword: EXPMID
-                title: UTC exposure mid time
-                blend_table: True
-                blend_rule: mean
+            title: UTC exposure mid time
+            type: number
+            fits_keyword: EXPMID
+            blend_table: True
+            blend_rule: mean
```

Comment:
Do we want to blend these in different ways?
```
mid_time:
  blend_rule: mean
```
and
```
mid_time_mjd:
  blend_rule: first
```
Do we even want to blend these?  These may not have any meaning in a combined image.

Reply:
In a combined image EXPSTART/MJD-BEG should give the starting time of the first blended exposure, EXPEND/MJD-END give the end time of the last blended exposure, and EXPMID/MJD-MID give the mid-point. So yes they can be useful in a blended image header.

Reply:
But I see that I mistakenly set all of the MJD-XXX keywords to blend_rule: first. I'll fix that. Thanks for catching!